ARCHITECTURE.md

```diff
@@ -261,8 +261,7 @@ two objects. The class is inherited by `_HOOMDBaseObject` to handle dependent
 relationships between operations in Python. The class defines *dependents* and
 *dependencies* of an object whose removal from a simulation (detaching) can be
 handled by overwriting specific methods defined in `_DependencyRelation` in
-`hoomd/operation.py`. See the interface of neighbor lists to pair potentials as
-an example of this in `hoomd/md/nlist.py` and `hoomd/md/pair/pair.py`.
```

Comment:
If this example is no longer relevant, can it be replaced with a different one?

Reply:
Unfortunately no. I don't think we use it anywhere else anymore. There are places we should/could use it, but we don't.